packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -132,11 +135,15 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
   constructor(props: IListProps<T>) {
     super(props);
 
+    initializeComponentRef(this);
+
     this.state = {
       pages: [],
       isScrolling: false
     };
 
+    this._async = new Async(this);
```

Comment:
Dispose this too

Reply:
Thank you, done in https://github.com/OfficeDev/office-ui-fabric-react/pull/9810/commits/74f0fa64252419bf31a307f4d1d510513aa40c4b.